### PR TITLE
Add warning if the user has not verified his/her email address yet

### PR DESCRIPTION
**Pull request by kaihendry**
Fixes: https://github.com/unee-t/frontend/issues/407

TODO:

* Resend verification email @nbiton (out of my depth) for users that have already registered.

Many thanks to @nbiton for helping me implement this.

### CODE REVIEW

**Comment by franck-boullier** [comment]
Unable to test ---> there are no Heroku version of the app available
unee-t-pr-414.herokuapp.com/

**Comment by kaihendry** [comment]
I do not know how to spawn a Heroku instance for testing. @nbiton 

Could that be documented somewhere like https://github.com/unee-t/frontend/wiki?

File: imports/ui/components/unverified-warning.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  isUserVerified: PropTypes.bool.isRequired,
+  email: PropTypes.string.isRequired
+
+}
+
+export default connect(
+  () => ({}) // Redux store to props
+)(createContainer(
+  () => {
+    console.log('emails', Meteor.user() && Meteor.user().emails)
```
Comment (nbiton) [inline imports/ui/components/unverified-warning.jsx]: Remove log

**Comment by franck-boullier** [comment]
This work as intended
Feel free to merge if you're OK with the code @nbiton 
image

**Comment by kaihendry** [comment]
Btw, it shouldn't be merged with "Resend email notification" implemented.

Any existing users without a verified email address could be stuck with this warning, otherwise?

**Comment by nbiton** [comment]
> Btw, it shouldn't be merged with "Resend email notification" implemented

@kaihendry with or without?

**Comment by nbiton** [comment]
@kaihendry Can you resolve the conflicts so we'll merge this? 
I think the resend of the verification email/ correcting the email address could be added separately

**Comment by kiatlim** [comment]
@nbiton @franck-boullier is this clearer? Addressing #450 here with change of wording. 
image

**Comment by nbiton** [comment]
@kiatlim What should happen after the email is sent? Should we change the link to a line that says "The email has been sent. Please check your inbox" or something similar?

**Comment by kiatlim** [comment]
Yeah “The email has been sent” should be succinct enough. Earlier line already has check your email. 

Cheers, 

Kiat Lim
+65 8 2233 956

**Pull request merged**